models/src/main/thrift/content/v1.thrift

```diff
@@ -1828,6 +1834,8 @@ struct RemovedContentResponse {
     8: required string orderBy
 
     9: required list<string> results
+
+    10: optional list<RemovedContent> resultsWithLastModified
```

Comment:
Maybe we can name this in way to provide more meaning? `removedContentResults` maybe?

Reply:
I'm assuming we'd always filter this out for normal search responses?

Reply:
This is on the `RemovedContentResponse` type, where currently `results` just gives you the removed content ids.
I'll make this new field hidden by default and add a query parameter that crier can send